src/pages/options/settings/MarketSettings.tsx

```diff
@@ -0,0 +1,35 @@
+import {Card, Divider, Flex, Space, Switch, Text, Title} from '@mantine/core';
+import {Control, Controller} from 'react-hook-form';
+import {SettingsType} from '../../../settings';
+
+interface MarketSettingsProps {
+    control: Control<SettingsType>;
+}
+
+export const MarketSettings = ({control}: MarketSettingsProps) => {
+    return (
+        <Card>
+            <Flex direction="column">
+                <Title order={4}>Market Settings</Title>
+                <Text fz="xs">Configure CSGOFloat Market Checker features in the market page and its listings.</Text>
+            </Flex>
+
+            <Space h="sm" />
+
+            <Flex gap="xl">
+                <Flex direction="column" w={500}>
+                    <Title order={6}>3D/Screenshot Buttons</Title>
+                    <Text fz="xs">
+                        Shows buttons under a listing to quickly view 3d model or screenshot of the specific item.
```

Comment:
If we define the options that _are_ possible in a schema (or just their own `Setting<name, type, data_type>`), then we can also define the description and auto-generate the entire page here.

Adding new settings would become super-simple, just adding the new defined `Setting` option and it all "hooks" up and allows toggling it here.

Reply:
i agree it would be really nice once we’ve fleshed out more settings and define specific data types but it feels like an over optimization right now

Reply:
would it be fair to add this once we have a third repeating data type for our settings

Reply:
actually at the very least i’ll component-ize this boolean setting field rn

Reply:
Sure, I think my solution is a bit of over-engineering for this. But yeah, let's prioritize making it simple to add new options fields.